content/posts/2020/04/libplanet-0.9.eng.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Libplanet 0.9 Released
+date: 2020-04-28
+authors: [hong.minhee]
+translators: [kidon.seo]
+---
+
+Hello Everyone! We have released the ninth minor version of [Libplanet], [Version 0.9][0.9.0].
+
+Libplanet is a common library that solves game implementation problems such as P2P communication and data synchronization when creating online multiplayer games that run on distributed P2P.
+
+From this version, Libplanet will be distributed in several NuGet packages. This is because Libplanet's externalities have expanded, making it too heavy to include some of the dependent libraries across all applications.
+
+This piece introduces the new NuGet packages and addresses key major changes in Version 0.9.
+
+
+[Libplanet]: https://libplanet.io/
+[0.9.0]: https://github.com/planetarium/libplanet/releases/tag/0.9.0
+
+
+[Libplanet.RocksDBStore]
+------------------------
+
+A newly added Nuget package, Libplanet.RocksDBStore includes `RocksDBStore` class, which is Lipblanet's [`IStore` interface][IStore] implemented as [RocksDB] backend. Through internal testing, `RocksDBStore` has shown to be about 10 times faster to write, 2 times faster to read, and thanks to compression, takes 15% less storage space than [`DefaultStore`][DefaultStore].
+
+Despite the advantages mentioned above, it might be difficult to use RocksDBStore on some platforms because the RocksDB native binary, written in C++, needs to be distributed together with an application.
```

Comment:
```suggestion
Despite the advantages mentioned above, it might be difficult to use `RocksDBStore` on some platforms because the RocksDB native binary, written in C++, needs to be distributed together with an application.
```